pkg/controller/cyclenoderequest/transitioner/transitions.go

```diff
@@ -431,14 +437,16 @@ func (t *CycleNodeRequestTransitioner) transitionCordoning() (reconcile.Result,
 			// Try to send the trigger, if is has already been sent then this will
 			// be skipped in the function. The trigger must only be sent once
 			if err := t.sendPreTerminationTrigger(node); err != nil {
-				return t.transitionToHealing(errors.Wrapf(err, "failed to send pre-termination trigger, %s is still cordononed", node.Name))
+				t.rm.LogEvent(t.cycleNodeRequest, "PreTerminationTriggerFailed", "failed to send pre-termination trigger to %v, err: %v", node.Name, err)
+				return t.transitionToHealing(errors.Wrapf(err, "failed to send pre-termination trigger to %s is still cordononed", node.Name))
```

Comment:
nit: can you revert `failed to send pre-termination trigger, %s is still cordononed`, the grammar is no longer correct.